addons/sourcemod/scripting/shop/helpers.sp

```diff
@@ -139,3 +139,12 @@ stock int Helpers_Math_Abs(int value)
 {
 	return (value ^ (value >> 31)) - (value >> 31);
 }
+
+/**
+ * 
+ */
+void Helpers_ClearArrayWithChatReason(ArrayList hArray, int client, const char[] str)
+{
+	delete hArray;
+	CPrintToChat(client, "%t", str);
+}
```

Comment:
Maybe extend this function prototype to something like `Handle hHndl, int client, const char[] szFormatString, any ...)` and rename in `Helpers_CloseHandleWithChatReason`?

Reply:
Good idea.